universal-application-tool-0.0.1/app/controllers/admin/QuestionController.java

```diff
@@ -59,6 +60,22 @@ public QuestionController(
             httpExecutionContext.current());
   }
 
+  @Secure(authorizers = Authorizers.Labels.UAT_ADMIN)
+  public CompletionStage<Result> show(Request request, Long id) {
```

Comment:
nit: can you use `questionId` instead of just `id`. It isn't immediately obvious what that is.

Reply:
Feel silly objecting to a nit, but so far the pattern in the controllers is that if everything in the file refers to the same object, we use `id`.  Program controller and Question controller both have only one ID, so I haven't done this.